ynput/ayon-nuke
# Repair action from validate correct asset context to fix the write node's filepath

## Changelog Description
This PR is to fix the bug of the validator for asset context not repairing the write nodes filepath as expected.


## Additional info
n/a


## Testing notes:

1. Launch Nuke
2. Change your folder path and task
3. Publish
4. Validate Correct Asset Context errors out
5. Click Repair
6. Should be fixing the filepath along with the correct context data.
